OCPBUGS-38891 - 4.14 RN for known issue - network outages (#81411)
openshift/openshift-docs: release_notes/ocp-4-14-release-notes.adoc

* There is a known issue in this release preventing access to the Web Terminal Operator after it has been installed. This issue will be fixed in a future {product-title} release. (link:https://issues.redhat.com/browse/OCPBUGS-14463[*OCPBUGS-14463*])

> * There is a known issue when upgrading the Cluster Network Operator from version 4.13 to 4.14.

Reviewer: ```suggestion
* A known issue exists when you upgrade the Cluster Network Operator from version 4.13 to 4.14.
```
Per IBM Style, avoid starting a sentence with the phrase "There is" or "There are".

Author: I was following the pattern here:
https://redhat-documentation.github.io/supplementary-style-guide/#_known_issue
Does the supplementary guide take precedence over IBM or the other way around?

The conversion to the new OVN Kubernetes interconnect multi-zone architecture can lead to packet drops that cause network outages of up to 300ms per node. 

> The impact is on East/West traffic in local gateway mode with `routingViaHost` set to `true`.

Reviewer: ```suggestion
The impact is on East/West traffic in local gateway mode with `routingViaHost` set to `true`. (link:https://issues.redhat.com/browse/OCPBUGS-38891[*OCPBUGS-38891*])
```

Author: Thanks for the review @lahinson 
I've included the link as you suggested
